recipes/quorum/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+pushd $SRC_DIR
+
```

Comment:
Can you add: `sed -i.bak '1 s|^.*$|#!/usr/bin/env perl|g' $PREFIX/bin/autoreconf` hopefully it will make it work.